doc/release.md

```diff
@@ -71,7 +71,8 @@ cherry-picked on the maintenance branch.
 - Bump the version and release by using `tbump X.Y-1.Z --no-push`. (For example:
   `tbump 2.3.5 --no-push`)
 - Check the result visually with `git show`.
-- Push the tag.
+- Open a merge request to run the CI tests for this branch
```

Comment:
😄 

Although `tests` did seem to pass on the branch?

Reply:
😅 No, the check on non default branches are limited it's only pre-commit that is launched (unless we open a MR).

Reply:
Sorry for not noticing/pointing out during review!